libs/shared/console-shared/src/lib/entrypoint-cmd-inputs/entrypoint-cmd-inputs.tsx

```diff
@@ -0,0 +1,66 @@
+import { Controller, useFormContext } from 'react-hook-form'
+import { InputText, InputTextArea } from '@qovery/shared/ui'
+
+export interface EntrypointCmdInputsProps {
+  className?: string
+  entrypointRequired?: boolean
+  cmdRequired?: boolean
+  imageEntryPointFieldName?: string
+  cmdArgumentsFieldName?: string
+}
+
+export function EntrypointCmdInputs(props: EntrypointCmdInputsProps) {
```

Comment:
With files you add on the **shared-console** we have mix between UI and Feature
For example `setting-resources` and `general-container-settings` has UI folder and no this file, can you harmonize this all files on the shared-console?

Same with `job-configure-settings` ; `job-general-setting` and others